lib/websockets.py

```diff
@@ -98,26 +107,26 @@ def run(self):
             method = r.get('method')
             result = r.get('result')
             if result is None:
-                continue    
+                continue
             if method == 'blockchain.scripthash.subscribe':
                 addr = r.get('params')[0]
```

Comment:
As I said on IRC, I am fairly certain this will fail here now, as you no longer give the address to the network.
`r.get('params')[0]` is now a scripthash

Reply:
Wait. I checked that. I still have your commit open: https://github.com/spesmilo/electrum/commit/8591587f9ed9e055edd0205ab45e9412e52c0f63
The issue is that, if the method is `scripthash.subscribe` the params contain an address not a scripthash. I must have misinterpreted the commit.

Reply:
the issue stems from if you call `network.subscribe_to_scripthash` then you will get back a scripthash as param, and if you call `network.subscribe_to_addresses` then you will get back an address